addons/binding/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/handler/SamsungTvHandler.java

```diff
@@ -133,39 +117,30 @@ public void channelLinked(ChannelUID channelUID) {
         }
     }
 
-    private synchronized void setPowerState(boolean state) {
-        powerState = state;
-    }
-
-    private synchronized boolean getPowerState() {
-        return powerState;
-    }
-
     @Override
     public void initialize() {
-        updateStatus(ThingStatus.OFFLINE);
+        updateStatus(ThingStatus.UNKNOWN);
 
         configuration = getConfigAs(SamsungTvConfiguration.class);
 
         logger.debug("Initializing Samsung TV handler for uid '{}'", getThing().getUID());
 
-        if (discoveryServiceRegistry != null) {
-            discoveryServiceRegistry.addDiscoveryListener(this);
+        nonUpnpRemoteControllerJob = scheduler.scheduleWithFixedDelay(this::checkCreateManualConnection, 0, 1,
+                TimeUnit.MINUTES);
+
+        if (StringUtils.isBlank(configuration.udn)) {
+            updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE, "Running without UPNP support, UDN is not configured");
+        } else if (upnpIOService == null) {
+            updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE, "Running without UPNP support, upn service is missing");
+        } else {
+            upnpIOService.registerParticipant(this);
         }
 
-        nonUpnpRemoteControllerJob = scheduler.scheduleWithFixedDelay(this::checkCreateManualConnection, 1, 1,
-                TimeUnit.MINUTES);
+        upnpPollingJob = scheduler.schedule(this::checkAndCreateServices, 0, TimeUnit.MILLISECONDS);
```

Comment:
you can use `submit` instead

Reply:
Thanks for spotting this crucial bug :-D

Reply:
It was called pollingJob but it was not a pollingJob